components/clm/src/main/decompInitMod.F90

```diff
@@ -967,8 +967,9 @@ subroutine decompInit_gtlcp(lns,lni,lnj,glcmask)
     deallocate(cstart, ccount)
     deallocate(pstart, pcount)
     if ( use_fates ) then
-       deallocate(coStart,coCount)
+       deallocate(coStart)
     endif
+    deallocate(coCount)
```

Comment:
`coCount` is allocated always, but was deallocated only when `use_fates=.true.`. Move outside of the if-stmt.